include/gpioMonitor.hpp

```diff
@@ -90,6 +92,9 @@ class GpioEventHandler
 
     // Inventory path of the FRU
     const std::string& m_inventoryPath;
+
+    // Command to bind or unbind the driver
+    std::string m_bindOrUnbindCmd;
```

Comment:
as the comment saying "command to bind the driver", why the name m_bindOrUnbindCmd,
should it be bindCmd ?

Reply:
done